builders/interoperability/xcm/xc20/xcm-erc20s.md

```diff
@@ -0,0 +1,338 @@
+
+The `transfer` function accepts the following parameters:
+
+- `currencyId` - the ID of the currency being sent via XCM. Different runtimes have different ways to define the IDs. In the case of Moonbeam-based networks, the following currency options exist:
+
+    - `SelfReserve` - refers to the native token
+    - `ForeignAsset` - refers to the asset ID of an [External XC-20](/builders/interoperability/xcm/xc20/xc20){target=_blank} (not to be confused with the XC-20 address)
+    - `LocalAssetReserve` - refers to the asset ID of a [Mintable XC-20](/builders/interoperability/xcm/xc20/mintable-xc20){target=_blank} (not to be confused with the XC-20 address)
+    - `Erc20` - refers to the contract address of an [ERC-20](/builders/interoperability/xcm/xc20/send-erc20s-xcm){target=_blank}
+
+    You'll be using `Erc20` as the `currencyId` for this guide and supplying the H160 Ethereum-style address of the ERC-20 contract
+
+- `amount` - the number of tokens that are going to be sent via XCM. Accepts a *uint128*
+- `dest` -  a multilocation to define the destination address for the tokens being sent via XCM. It supports different address formats, such as 20 or 32-byte addresses (Ethereum or Substrate)
+- `destWeightLimit` - an enum that represents the maximum amount of execution time you want to provide in the destination chain to execute the XCM message being sent. The enum contains the following options:
+
+    - `Unlimited` - allows for the entire amount used for gas to be used to pay for weight
+    - `Limited` - limits the amount used for gas to a particular value
+    
+    If not enough weight is provided, the execution of the XCM will fail, and funds might get locked in either the Sovereign account or a special pallet. **It is important to correctly set the destination weight to avoid failed XCM executions**
+
+Now that you have a basic understanding of the `transfer` function and the parameters it accepts, you can build and send the XCM message.
+
+### Use the Polkadot.js API to Send an ERC-20 Cross-Chain {: #use-polkadotjs-apps }
+
+This section of the guide will show you how to build and send an XCM message to transfer an ERC-20 cross-chain using the Polkadot.js API. This is purely for example purposes only. Before sending assets cross-chain, make sure that the asset you're sending is [registered in the target chain](#register-an-erc-20-on-another-chain); otherwise, the call will result in the loss of the assets.
+
+Since you'll be interacting with the `transfer` function of the X-Tokens Pallet, you'll need to gather the arguments for the `currencyId`, `amount`, `dest`, and `destWeight`. For this example, you can send 1 ERC-20 token to an account on the relay chain:
+
+- The `currencyId` for an ERC-20 token has the following parameters:
+
+    |    Parameter    |         Value         |
```

Comment:
I've been thinking about this and I think tomorrow I'm going to remove these tables. It made sense to have tables like these when using Polkadot.js Apps, but now I think it just makes more sense to focus on the JavaScript code